java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_2/Tree.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class Tree {
+
+    class Node {
+    int value;
+    Node leftNode;
+    Node rightNode;
+
+    Node(int value) {
+        this.value = value;
+        rightNode = null;
```

Comment:
leftNode and rightNode will have a value null by default, it's not necessary to do this in a constructor

Reply:
Already remove this redundancy.